From 8e1288d06d53ebec1a8d7947a4813c346b132849 Mon Sep 17 00:00:00 2001
From: Benjamin Tissoires <benjamin.tissoires@redhat.com>
Date: Tue, 26 Aug 2014 17:18:52 -0400
Subject: [input] alps: change secondary device's name

Message-id: <1409073549-31548-2-git-send-email-benjamin.tissoires@redhat.com>
Patchwork-id: 89234
O-Subject: [PATCH RHEL7 01/18] Input: ALPS - change secondary device's name
Bugzilla: 1107819
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1107819
Upstream: since v3.13
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7882179
Tested by me on a Toshiba Portege R30 with a v7 Alps touchpad

commit 9354f263383975ed5e11eb5a9cf21f2596f95be2
Author: Yunkang Tang <yunkang.tang@cn.alps.com>
Date:   Tue Oct 15 23:50:38 2013 -0700

    Input: ALPS - change secondary device's name

    Change the dev2's name from "PS/2 Mouse" to "ALPS PS/2 Device".

    Signed-off-by: Yunkang Tang <yunkang.tang@cn.alps.com>
    Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>

Signed-off-by: Benjamin Tissoires <benjamin.tissoires@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/input/mouse/alps.c b/drivers/input/mouse/alps.c
index 7c5d72a..ca7a26f 100644
--- a/drivers/input/mouse/alps.c
+++ b/drivers/input/mouse/alps.c
@@ -1792,7 +1792,7 @@ int alps_init(struct psmouse *psmouse)
  snprintf(priv->phys, sizeof(priv->phys), "%s/input1", psmouse->ps2dev.serio->phys);
  dev2->phys = priv->phys;
  dev2->name = (priv->flags & ALPS_DUALPOINT) ?
-       "DualPoint Stick" : "PS/2 Mouse";
+       "DualPoint Stick" : "ALPS PS/2 Device";
  dev2->id.bustype = BUS_I8042;
  dev2->id.vendor  = 0x0002;
  dev2->id.product = PSMOUSE_ALPS;
-- 
1.7.1